packages/devtools_app/lib/src/app.dart

```diff
@@ -215,7 +213,7 @@ class DevToolsAppState extends State<DevToolsApp> with AutoDisposeMixin {
                 .where((p) => embed && page != null ? p.screenId == page : true)
                 .where((p) => !hide.contains(p.screenId))
                 .toList();
-            if (tabs.isEmpty) return child;
+            if (tabs.isEmpty) return child!;
```

Comment:
I don't know enough about ValueListenableBuilder to suggest what needs to be returned here. It looks like child may end up being null and causing an issue here though so we may need to find a way to return a blank widget or something agreeable to the builder.